buildPackages.cmd

```diff
@@ -0,0 +1,24 @@
+@echo off
+setlocal
+
+set packagesLog=buildPackages.log
+echo Running buildPackages.cmd %* > %packagesLog%
+
+if /I [%1] == [/?] goto Usage
+if /I [%1] == [/help] goto Usage
+
+echo msbuild.exe %~dp0buildPackages.proj %* /nologo /v:minimal /flp:v=detailed;Append;LogFile=%packagesLog% >> %packagesLog%
+call msbuild.exe %~dp0buildPackages.proj %* /nologo /v:minimal /flp:v=detailed;Append;LogFile=%packagesLog%
```

Comment:
My bad when doing a rebase before pushing the change. I updated the file. Thanks Davis